modules/SofaConstraint/LCPConstraintSolver.cpp

```diff
@@ -397,7 +397,7 @@ void LCPConstraintSolver::build_Coarse_Compliance(std::vector<int> &constraint_m
 
     _Wcoarse.clear();
 
-    dmsg_error_when(sizeCoarseSystem==0) <<"no constraint" ;
+    dmsg_error_when( (sizeCoarseSystem==0) ) <<"no constraint" ;
```

Comment:
What is the warning needing this? Maybe it could be done directly in the macro to preserve a good looking syntax for the user :-)

Reply:
https://ci.inria.fr/sofa-ci/job/ubuntu_clang-3.8_default/587/warnings11Result/
> equality comparison with extraneous parentheses